src/node_crypto.cc

```diff
@@ -1974,7 +1974,78 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
-// TODO(indutny): Split it into multiple smaller functions
+// Put issuer inside the object
+static void AddIssuer(X509** cert,
+                      const STACK_OF(X509)* const peer_certs,
+                      Local<Object>* const info,
+                      Environment* const env) {
+  *cert = sk_X509_delete(peer_certs, 0);
+  while (sk_X509_num(peer_certs) > 0) {
```

Comment:
Unrelated observation: I think this could just be `for (;;) {` - the same check is effectively done at the end of the loop when `i == 0`.

Reply:
Nice, I'll take a closer look at this.